cartoframes/io/carto.py

```diff
@@ -93,6 +96,11 @@ def to_carto(dataframe, table_name, credentials=None, if_exists='fail', geom_col
         ValueError: if the dataframe or table name provided are wrong or the if_exists param is not valid.
 
     """
+    def estimate_csv_size(gdf):
```

Comment:
I would move `estimate_csv_size` to ` utils.utils`, also `compute_copy_data` because we are using it in different places.

Reply:
I haven't moved it to `utils.utils` because that would mean that `utils.utils` would depend on `io.managers.context_manager` and as that file already depends on `utils.utils` that would create a circular dependency

Reply:
Oh, I'll move it to `utils.columns` then or at least out of the `to_carto` function.

Reply:
The `utils.columns` file has the same problem with the circular dependencies.

I have tried extracting the method and all of it's dependencies to `utils.utils` or `utils.columns` but in that case I find circular dependencies between `utils.utils` and `utils.columns`

Finally I have extracted the `estimate_csv_size` method to the `carto.py`  file but outside the `to_carto` method